server/infrastructure/auth_repository.py

```
@@ -0,0 +1,132 @@
+            )
+        except auth.UserNotFoundError:
+            return None
+        except Exception:
+            return None
+    
+    async def create_custom_token(self, user_id: str) -> str:
+        """Create custom token for user"""
+        try:
+            custom_token = auth.create_custom_token(user_id)
+            return custom_token.decode('utf-8')
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)